website/source/docs/auth/github.html.md

```diff
@@ -68,4 +72,15 @@ Success! Data written to: auth/github/map/teams/owners
 ```
 
 The above would make anyone in the "owners" team a root user in Vault
-(not recommended).
+(not recommended). Note: the team ("owners") is case-sensitive.
```

Comment:
Ah, okay (I didn't necessarily think this was an issue.) Please let me know if you'd like me to change any of this (or if you just want to ignore it). I thought the most important part was noting what scope is required for the PAT to make this all work.
Thanks.